lib/components/ms-spinner/MsSpinner.vue

```diff
@@ -8,28 +8,23 @@
     >
       {{ $msTranslate(title) }}
     </ion-text>
-    <vue3-lottie
-      class="container-spinner"
-      :animation-data="SpinnerJSON"
-      :height="height || 24"
-      :width="width || 24"
-      :speed="speed || 1"
-      :loop="true"
-    />
+    <div class="container-spinner"></div>
   </div>
 </template>
 
 <script setup lang="ts">
-import SpinnerJSON from '@lib/assets/spinner.json';
 import { Translatable } from '@lib/services/translation';
 import { IonText } from '@ionic/vue';
+import { onMounted } from 'vue';
 
-defineProps<{
+const props = defineProps<{
   title?: Translatable;
-  height?: number;
-  width?: number;
-  speed?: number;
+  size?: number;
 }>();
+
+onMounted(() => {
```

Comment:
Doesn't it work with just
```
<script>
const props = defineProps<{
  size?: number;
}>

const spinnerSize = ref(`${props.size || 24}px`);
</script>

<style>
  width: v-bind(size);
</style>
```
? I find it a bit easier to read without the call to onMounted, and I'm not sure how scoped the `--ms-spinner-size` property is (does `document.documentElement` refers to the component or `<html>` here?).

Reply:
it's the same method we use to set css variables in SideBarMenu.Page

Reply:
I replaced, it's better